app/test/ui/widgets/custom_search_bar_test.dart

```diff
@@ -0,0 +1,60 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:medi_support/ui/widgets/custom_search_bar.dart';
+
+void main() {
+  CustomSearchBar customSearchBarWithoutQuery = CustomSearchBar(
```

Comment:
**suggestion:** Move into the tests, the test should not be dependet on a variable outside the test. exceptions are variables reset by "setup(...)"